ext/openssl/tests/openssl_pkcs7_verify_basic.phpt

```diff
@@ -36,6 +42,11 @@ if (file_exists($contentfile)) {
 	echo "true\n";
 	unlink($contentfile);
 }
+
+if (file_exists($pkcsfile)) {
```

Comment:
Ok, maybe the others should be moved there too. But I would rather see that in a separate PR.